hera_mc/tests/test_parts.py

```diff
@@ -81,9 +79,10 @@ def test_update_part(parts, capsys):
     assert 'does not exist as a field' in captured.out.strip()
     data = [[parts.test_part, parts.test_rev, 'hpn_rev', 'Z']]
     cm_partconnect.update_part(parts.test_session, data)
-    dtq = Time('2017-07-01 01:00:00', scale='utc')
-    located = parts.cm_handle.get_part_dossier(
-        hpn=[parts.test_part], rev='Z', at_date=dtq, exact_match=True)
+    dtq = Time('2019-09-01 01:00:00', scale='utc')
+    located = parts.cm_handle.get_dossier(
+        hpn=[parts.test_part], rev=None, ports=None, at_date=dtq, exact_match=True)
+    print(located)
```

Comment:
should this print statement be taken out?

Reply:
Yes

Reply:
Yes